packages/create-toolpad-app/src/generateProject.ts

```diff
@@ -139,24 +100,66 @@ export default function generateProject(
   `;
 
   const themeContent = `
-  "use client"
-  import { Roboto } from "next/font/google";
+  "use client";
+  import { Inter } from "next/font/google";
   import { createTheme } from "@mui/material/styles";
-  
-  const roboto = Roboto({
+
+  const inter = Inter({
     weight: ["300", "400", "500", "700"],
     subsets: ["latin"],
     display: "swap",
   });
-  
-  const theme = createTheme({
+
+  const baseTheme = createTheme({
```

Comment:
It would be ideal to have a design to match this theme to, I kind of made it look like the other template we have but not always sure about the right colors for example.